pkg/engine/policyContext.go

```diff
@@ -20,4 +21,6 @@ type PolicyContext struct {
 	Client *client.Client
 	// Contexts to store resources
 	Context context.EvalInterface
+	// Config handler
+	Config config.Interface
```

Comment:
With the current implementation, both engine.mutate and engine.validate take the config interface, but for the same admission request, the exclude list should be the same. Can we add `excludeList []string` instead of adding the interface to policyContext?
One other benefit is that this will remove the dependency on config package, as CLI also re-use the engine code, take the static list could reduce complexity potentially.